_pages/dev/styling-and-page-layout/directionality.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Directionality
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 2.1 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+The directionality feature provides support for bidirectional text and layout. You can configure Spartacus to use a left-to-right (LTR) orientation, or a right-to-left (RTL) orientation.
+
+Directionality is driven by language. Many languages are read from left to right, but well known examples, such as Arabic and Hebrew, are read from right to left.
+
+In Spartacus, the direction of the UI reflects the active language, so that directionality can work in a multi-directional experience. If your storefront contains both LTR and RTL languages, the active language is used to detect the direction automatically.
+
+## Breaking Changes
+
+The changes made to the DOM and CSS are considered breaking changes, so the directionality feature can only be used if you enable it with the 2.1 feature flag and the 2.1 CSS version.
```

Comment:
can we add a link to the css versioning page, give that it's also a new feature with 2.1?

Reply:
Done 👍